### PR TITLE
update numeraire msg

### PR DESCRIPTION
**Pull request by EliLazarus**
I think this is clearer.
For issue #162

### CODE REVIEW

**Comment by mitchphillipson** [comment]
I included a message about unfixing the variable. I've debated unfixing it just after the solve. There is currently some logic remembering the old numeraire and unfixing it. 

I want your opinion. This _should_ be an easy change and it makes a little sense for this PR. 

Let me know.

**Comment by mitchphillipson** [comment]
Here is an example showing that the numeraire is fixed

```julia
using MPSGE

M = MPSGEModel()

@sectors(M, begin
    A
end)

@commodities(M, begin
    PA
    PB
end)

@consumer(M, RA)

@production(M, A, [s=0, t=0], begin
    @output(PA, 1, t)
    @input(PB, 1, s)
end)

@demand(M, RA, begin
    @final_demand(PA, 1)
    end,begin
    @endowment(PB, 1)
end)

solve!(M)

is_fixed(RA)
```

It should be fairly simple to undo this. I think I did this before we started using a `direct_model`. A standard JuMP model does not allow you get values if you change the model in any way (like fixing a variable).

**Comment by mitchphillipson** [comment]
I've made the changes. It was very easy, except for the tests. I think it's better this, I didn't care for fixing a variable and having it be kind of hidden.

File: src/build.jl [hunk 1]
```diff
@@ -258,35 +258,27 @@ function solve!(m::AbstractMPSGEModel; kwargs...)
...
+    end
 
     if !m.silent
         output = "\n\nSolver Status: $(termination_status(jm))\nModel Status: $(primal_status(jm))"
 
         if !isnothing(consumer)
-            output *= "\n\nDefault price normalization using income for $consumer - This value is fixed to $(value(consumer)).\n"* 
-                      "Unfix with unfix($consumer)."
-            #unfix(consumer)
+            output *= "\n\nDefault price normalization using income for $consumer as numeraire, with the value of $(value(consumer)).\n"
```
Comment (mitchphillipson) [inline src/build.jl:280]: These changes are great. I've made more now

**Comment by EliLazarus** [comment]
Ok, great. Getting rid of the numeraire field is cleaner. And all works. Nice little PR.

**Pull request merged**